__tests__/cli.test.ts

```diff
@@ -259,13 +298,16 @@ class CLIMock {
     );
     const dataListener = data => {
       this.data = data.toString();
+      // Uncomment the line below if the process is blocked and you need to see its output
+      // console.warn(this.data);
       this.chunks.push(...this.data.split('\n').filter(Boolean));
       if (this.waitForPromise && this.data.includes(this.waitForText)) {
         this.process.stdout.off('data', dataListener);
         this.waitForPromise();
       }
     };
     this.process.stdout.on('data', dataListener);
+    this.process.stderr.on('data', dataListener);
```

Comment:
This is inccorrect, Any data in stderr, should exit the process which is handled here - https://github.com/elastic/synthetics/blob/6cf145200e73d6c127cf12e34be36294504e76df/__tests__/cli.test.ts#L313